.vscode/settings.json

```diff
@@ -53,5 +52,6 @@
             "front_matter_title": ""
         }
     },
-    "editor.inlayHints.enabled": "offUnlessPressed"
+    "editor.inlayHints.enabled": "offUnlessPressed",
+    "dotnet.defaultSolution": "FSharp.Compiler.Service.sln"
```

Comment:
omnisharp.defaultLaunchSolution -> dotnet.defaultSolution - this change is on purpose?

Reply:
It's automated by extension itself, so yes.